muted-tests.yml

```diff
@@ -339,6 +339,9 @@ tests:
 - class: org.elasticsearch.threadpool.SimpleThreadPoolIT
   method: testThreadPoolMetrics
   issue: https://github.com/elastic/elasticsearch/issues/108320
+- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
```

Comment:
did you mute this in main already? We should make sure this is actually muted in the appropriate PR and then update this branch once that mute is merged.

Reply:
This is going away with the next merge with the lucene_snapshot branch, I just cherry-picked it to pass CI